### DISCUSSION TITLE
Allurectl config folder

### DISCUSSION THREAD

**Discussion by johnny550** [allure-framework/allure2#2821] [TestOps]
Hello.
I am running allurectl in a read-only system. The thing is the cli seems to create a folder in my user home folder.
Here is what I did

`
export ALLURE_ENDPOINT=...
export ALLURE_TOKEN=...

allurectl project get ###
`
Right after running that command allurectl attempts to create the folder at ~/.allure.
Is there a way to make allurectl stop doing that? All the necessary env vars are given, so I assume there should be no need for allurectl to try creating a folder that would hold a config file that will not be created.

Can you guys please help?
Cheers

**Comment by cheshi-mantu** [comment]
@johnny550 hi!
Could you please describe your use case in more detail using our dedicated help desk at https://help.qameta.io/? Please use your work email address when registering on the portal.
cheers
